src/libraries/Microsoft.Extensions.Configuration.CommandLine/src/Microsoft.Extensions.Configuration.CommandLine.csproj

```diff
@@ -1,18 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(DefaultNetCoreTargetFramework);net461;$(NetFrameworkCurrent)</TargetFrameworks>
-    <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
+    <!-- Use targeting pack references instead of granular ones in the project file. -->
```

Comment:
what do you mean by this, I didn't get what are you referring to with `granular`.

Reply:
This means that we use the default targeting pack references (all) for this project instead of listing the individual references in the project file. We don't want to do this for libs that are part of the shared framework or for OOBs where dependencies matter but for some of the Microsoft.Extensions projects it doesn't matter. I'm experimenting with this feature in these few projects.